src/test/java/roomescape/reservation/domain/ReserverNameTest.java

```diff
@@ -0,0 +1,26 @@
+package roomescape.reservation.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+class ReserverNameTest {
+
+    @Test
+    @DisplayName("예약자 이름은 비어있을 수 없다")
+    void cannotEmptyReserverName() {
+        // given
+        // when
+        // then
+        assertAll(() -> {
```

Comment:
별도의 given 이 없다면 다음과 같이 표현하셔도 좋아요
가독성을 위한 것이니까요

```suggestion
        // when then
        assertAll(() -> {
```

Reply:
개인적으로는 given, when, then을 모두 명시하는 방식이 이후 given에 내용이 추가될 때 더 유연하게 대응할 수 있었고, 특히 given에 아무 내용이 없다는 점도 명확히 드러나 스스로 코드를 점검해 보는 계기가 되었습니다.
라이브 템플릿을 활용해 작성하다 보니 코드를 입력하는 데에는 큰 불편이 없었지만, 파일 크기가 커짐에 따라 유지 보수성이 떨어지고, 가독성이 떨어져서 리뷰하실 때 불편함을 드릴 수 있겠다는 생각도 들었습니다.
앞으로는 권장해 주시는 방식대로 표현해겠습니다!